### PR TITLE
BUG: add metatags to template

### PR DESCRIPTION
**Pull request by sunnysideup**
## Description

image

The `Metatags` listed here return nothing. We should either remove them or link to SiteTree. I have linked to SiteTree here. 

Also, we should be consistent and always talk about `MetaTags`  with two capitals. 

## Issues

- #

## Pull request checklist

- [ ] The target branch is correct
    - See [picking the right version](https://docs.silverstripe.org/en/contributing/code/#picking-the-right-version)
- [ ] All commits are relevant to the purpose of the PR (e.g. no debug statements, unrelated refactoring, or arbitrary linting)
    - Small amounts of additional linting are usually okay, but if it makes it hard to concentrate on the relevant changes, ask for the unrelated changes to be reverted, and submitted as a separate PR.
- [ ] The commit messages follow our [commit message guidelines](https://docs.silverstripe.org/en/contributing/code/#commit-messages)
- [ ] The PR follows our [contribution guidelines](https://docs.silverstripe.org/en/contributing/code/)
- [ ] Code changes follow our [coding conventions](https://docs.silverstripe.org/en/contributing/coding_conventions/)
- [ ] This change is covered with tests (or tests aren't necessary for this change)
- [ ] Any relevant User Help/Developer documentation is updated; for impactful changes, information is added to the changelog for the intended release
- [ ] CI is green

### CODE REVIEW

**Comment by GuySartorelli** [comment]
Hi there,

Thank you for your contribution.
I notice that you didn't follow the pull request template - I've added that template back in to your pull request description. Please fill in all of the relevant information and follow the documented contribution guidelines.

**Comment by sunnysideup** [comment]
I will do my best.  I know I am one of the laziest people when it comes to making pull requests. I do plenty for the open source community, but I lack the time to do all of this stuff - so I double-muchly appreciate all your help!

I would never mind if someone else would take my code and add it with the required bits to make it their commit. 

When I have an opportunity I will finish this.

**Comment by sunnysideup** [comment]
@GuySartorelli  - do you think it is worthwhile adding?

**Comment by GuySartorelli** [comment]
> do you think it is worthwhile adding?

If you mean the template information, then yes it is worthwhile. Using this template ensures:
1. The person making the contribution (in this case you) has read and followed the contribution guidelines
2. The commits are all relevant
3. The commit messages are all formatted correctly
3. There are tests if appropriate
3. Docs are updated as appropriate
4. There is an issue for us to track and add any discussion to if needed

Basically, it makes it a lot easier to manage contributions to [all of the supported modules](https://docs.silverstripe.org/en/project_governance/supported_modules/)
